config/webhook/manifests.yaml

```
@@ -7,7 +6,7 @@ metadata:
   name: mutating-webhook-configuration
 webhooks:
 - admissionReviewVersions:
-  - v1beta1
+  - V1
```

Here as well.  (The rest of this file also requires the same changes)
```suggestion
  - v1
```